src/main/java/net/dv8tion/jda/api/interactions/callbacks/IPremiumRequiredReplyCallback.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.interactions.callbacks;
+
+import net.dv8tion.jda.api.requests.restaction.interactions.PremiumRequiredCallbackAction;
+
+import javax.annotation.CheckReturnValue;
+import javax.annotation.Nonnull;
+
+/**
+ * Interactions which allow sending the "Premium required" interaction response.
+ *
+ * <p>Replying with {@link #replyWithPremiumRequired()} will automatically acknowledge this interaction.
+ */
+public interface IPremiumRequiredReplyCallback extends IDeferrableCallback
```

Comment:
Can you also update the docs for [Interaction](https://docs.jda.wiki/net/dv8tion/jda/api/interactions/Interaction.html) with this new reply type